.nvmrc

```diff
@@ -0,0 +1 @@
+20.13.1
```

Comment:
Looks like `.yarn/install-state.gz` is part of git tracking when it should be skipped as per [yarns docs](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored)

Reply:
Ah, wonder why it wasn't originally part of git ignore. I'll fix that.

Reply:
I don't think the dev deps in this repo were a recent addition, so was probably just missed